extras/caller-log.rst

```diff
@@ -14,15 +14,65 @@ View and manage call logs from the **phone** panel.
    This feature is **optional**;
    if you don’t see it in the main menu,
    that means your administrator hasn’t enabled it yet.
-   Administrators can learn more
-   `here <https://admin-docs.zammad.org/en/latest/system-integrations.html#integrations-for-phone-systems>`_.
+   Administrators can learn more on our `admin documentation`_.
+
+.. _admin documentation:
+   https://admin-docs.zammad.org/en/latest/system/integrations.html#integrations-for-phone-systems
 
 .. hint::
-   🏢 The caller log shows all incoming and outgoing calls **for the entire team**.
+   🏢 The caller log shows all incoming and outgoing calls
+   **for the entire instance**. The number of entries shown depends on the
+   configuration your admin chose.
+
+The caller log offers a lot more than just the last call entries.
+If your administrator configured "Phone Extension to Agent Mapping", Zammad
+will also help you during answering calls.
+
+   New Ticket dialogue
+      Zammad will open a new ticket dialogue if:
+
+         * it's able to either guess a single user (see "maybe" entries below)
```

Comment:
Only one entry for "maybe", not entries. I suggest to call it **"Maybe" note**, as the header of the box also says "Note".

Reply:
After internal discussion the main issue was the way we reference the "note" about maybe entries further down the documentation page. This will be addressed by commit 302c34752c55f8b80bc1e102f12e2b2abbe29237